indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -657,6 +658,12 @@ public void run()
                 shouldProcess
             );
 
+            // Emit the processed bytes metric
+            final ServiceMetricEvent.Builder metricBuilder = new ServiceMetricEvent.Builder();
+            IndexTaskUtils.setTaskDimensions(metricBuilder, task);
+            toolbox.getEmitter().emit(
+                metricBuilder.setMetric("ingest/processed/bytes", rowIngestionMeters.getProcessedBytes()));
```

Comment:
Shouldn't this metric be emitted once per task?
The other task types seem to be doing that.